src/bloaty.proto

```diff
@@ -98,3 +101,12 @@ message Regex {
   optional string pattern = 1;
   optional string replacement = 2;
 }
+
+// Domain size capacities for a label in a data source.  When provided, the
+// output will display sizes as a percentage of this capacity.  Useful for
+// binaries in which there is a limit to how much space can be used.
+message LabelCapacity {
+  optional string label = 1;
```

Comment:
Perhaps mention in a comment that this must be an exact match.
Maybe drop a TODO to consider making this a regex.